### ISSUE TITLE
Add Kivy language support

### ISSUE THREAD

**Issue by amir511** [Microsoft/vscode#28491]
**Kivy is an open source framework for graphical user interface creation with Python language**
It is a very promising framework, it is multiplatform and supports multi touch devices
Kivy Simplifies the definition of UI through simple syntax called kivy lang or kv
example:
![image](https://user-images.githubusercontent.com/18488957/27024009-4e0efe12-4f55-11e7-844e-0b7fa808162b.png)
Please add the support of this language to the awesome VS code!
More details [here](https://kivy.org/docs/api-kivy.lang.html) about the kivy syntax.
Thanks.

**Comment by Simbiosis78** [comment]
Yes please! Kivy support on VSCode.

**Comment by BattleBas** [comment]
I started a project to create a extension that supports the Kivy language [here ](https://github.com/BattleBas/kivy-vscode)

So far it just offers some basic syntax highlighting and a few code snippets for defining widgets.
Still needs a lot of work, but thought this might help get things started.

Just for fun I also published it to the extension marketplace [here](https://marketplace.visualstudio.com/items?itemName=BattleBas.kivy-vscode)

Note: Hopefully this is the right place to post this, thanks!
